src/main/java/Cross.java

```diff
@@ -0,0 +1,48 @@
+public class Cross {
+    private final int posX;
+    private final int posY;
+    private final Dot dot;
+
+    private Cross(int posX, int posY, boolean left, boolean current) {
+        this.posX = posX;
+        this.posY = posY;
+        this.dot = new Dot(left, current);
```

Comment:
```suggestion
        this(posX, posY, new Dot(left, current));
```
부 생성자가 주 생성자를 호출해 구현할 것을 추천